### PR TITLE
[WIP]Call-for-Blogs: Introduction

### PR DESCRIPTION
**Pull request by NitishKumar06**
Partial Fixes:#34355

---

_Update:_ Ater going through the various blog types of Kubernetes, I have decided to push small changes with each PR (squash them at last) such that it'll be easier for the reviewers to review as well as lil easy for me to add those suggested changes

### CODE REVIEW

File: content/en/blog/call-for-blogs.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Call for blog contributions
+
+This is a constantly updated list of the blogs that the sig-docs is interested in.
```
Comment (divya-mohan0209) [inline content/en/blog/call-for-blogs.md:3]: We should first set the context by stating which blog we're referring to. There are two, and only https://kubernetes.io/blog/ is within the remit of SIG Docs. It is one of our subprojects with its own Slack channel and set of reviewers/approvers.

File: content/en/blog/call-for-blogs.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+This will list the blogs along with abstract that can be furthre used for gathering information.
```
Comment (divya-mohan0209) [inline content/en/blog/call-for-blogs.md:5]: This line should be inviting folks and asking them to reach out to us if they have any questions (on Slack) before they submit a PR to k/website. WDYT?

File: content/en/blog/call-for-blogs.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+---
```
Comment (divya-mohan0209) [inline content/en/blog/call-for-blogs.md:7]: When someone looks at the list below, what do we want them to understand?
From the submitted issue and the explanation provided on Slack, I assume we want the users to know about the different types of blogs we publish or that they can submit content for. If that's the case, the one-line description provided against each listing is inadequate.

**Review by divya-mohan0209 — DISMISSED** [review]
Overall review: 
- We do not issue a call-out for the release announcement, features, and deprecations/major changes blogs. The Release Docs & comms team manage their publishing. However, if we - are talking about the different kinds of blogs that the subproject publishes, we should retain one/two examples.
- You will need to include more examples of user-submitted examples by going through the blog listing published previously.

File: content/en/blog/call-for-blogs.md (hunk 1 above)
Comment (NitishKumar06) [reply content/en/blog/call-for-blogs.md:3]: Could you provide some resources that I should go with to learn about those two types? I assumed there's only one type, Hence I referred it here.

File: content/en/blog/call-for-blogs.md (hunk 3 above)
Comment (NitishKumar06) [reply content/en/blog/call-for-blogs.md:7]: Here's what I concluded from the issue:

As a [reference link](https://github.com/keras-team/keras-io/blob/master/call_for_contributions.md) provided on the issue, the blogs were selected with an intent to provide a little description of the _important blogs_.  Hence I listed some which include v1.26 features, etc. Do you think there's a need to change the number of blogs mentioned?

File: content/en/blog/call-for-blogs.md (hunk 2 above)
Comment (NitishKumar06) [reply content/en/blog/call-for-blogs.md:5]: From the issue, I'm thinking the introductory line will focus on mentioning a detail about some important blogs that most of the users are looking forward to. For eg. 

Folks would be interested to know more about Kubernetes:v1.26 release features - why it's an electrifying release?

However, the last line of the introduction can mention reaching out to slack for queries and references on how to submit blogs to Kubernetes. WDYT @divya-mohan0209 about this?

File: content/en/blog/call-for-blogs.md (hunk 1 above)
Comment (divya-mohan0209) [reply content/en/blog/call-for-blogs.md:3]: I mentioned it above, @NitishKumar06.
There is a blog on the Kubernetes Documentation website - https://kubernetes.io/blog.
There is another blog on the Kubernetes Contributor Community Website - https://www.kubernetes.dev/blog/

The first is the one within our remit.

File: content/en/blog/call-for-blogs.md (hunk 2 above)
Comment (divya-mohan0209) [reply content/en/blog/call-for-blogs.md:5]: Could you PR the changes in and then we'll see if it fits in? I am not able to understand what you're trying to convey.

File: content/en/blog/call-for-blogs.md (hunk 3 above)
Comment (divya-mohan0209) [reply content/en/blog/call-for-blogs.md:7]: So, let me rephrase this in a way you'd understand better. Release blogs, feature blogs, etc., are written by folks within the community who are working on the releases and features. They are submitted as a part of the release, and we DO NOT invite contributions externally for them, i.e., they are being worked on by active contributors within the community. 
I am unsure how the Keras project functions, but to write a call-out, you need to tailor the call-out specific to our community to make it relevant. Which is why, it is okay to include one/two release blog examples in the call-out but not have an entire listing about release-related blogs since there are other types of blogs we invite contributions for.

Comment (divya-mohan0209) [reply content/en/blog/call-for-blogs.md:7]: To understand the other types, you will need to go beyond the surface level of the first page for blogs and do a bit of research.

File: content/en/blog/call-for-blogs.md (hunk 1 above)
Comment (sftim) [reply content/en/blog/call-for-blogs.md:3]:
```suggestion
This is a list of the blog topics that the Kubernetes project is interested in.
```

- We don't “constantly update” any of our blog articles.
- If we did make a commitment to keep a page updated, I think it'd be something within https://k8s.dev/
- or possibly within https://kubernetes.io/docs/contribute/

File: content/en/blog/call-for-blogs.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Call for blog contributions
```
Comment (sftim) [inline content/en/blog/call-for-blogs.md:1]: This is not how to start a blog article. Check the existing articles to see how it's done.

**Review by sftim — COMMENTED** [review]
Thanks for starting this.

I like that you're keen to do this; however, the overall direction doesn't seem right.

**Comment by divya-mohan0209** [comment]
Hello @NitishKumar06 any update on this, please?

**Comment by NitishKumar06** [comment]
Hey @divya-mohan0209 , I'll push the changes once I'm done with the research, and let you know.

**Comment by sftim** [comment]
This PR is still marked as a work in progress. Is that right?

**Comment by NitishKumar06** [comment]
> This PR is still marked as a work in progress. Is that right?

Yes! Rather than putting out the entire content very first, I decided to go with producing a small chunk of final content that will help reviewers to make the review process easier and continue further once the review has been given.

**Comment by sftim** [comment]
“Work in progress” means, pretty much, _not_ ready for review.

**Comment by kbhawkey** [comment]
Hello @NitishKumar06 .
Checking if you are actively working on this pull request or if I can close it?
Thanks
